src/index.ts

```diff
@@ -153,7 +153,10 @@ export default function createIntegration(args?: Options): AstroIntegration {
           fs.writeFileSync(
             pth,
             contents.replace(
-              `import { serveFile, fromFileUrl } from '${DENO_IMPORTS_SHIM}';`,
+              new RegExp(
+                String
+                  .raw`import \{ serveFile, fromFileUrl \} from ['"]${DENO_IMPORTS_SHIM}['"];`,
+              ),
```

Comment:
I think more reliable and cleaner way is to modify 
DENO_IMPORTS_SHIM to include full import

Then you can do 
```ts
contents.replace(
  DENO_IMPORTS_SHIM,
  DENO_IMPORTS
)
```

Reply:
`DENO_IMPORTS_SHIM` is used in other places to push the dependency.

Reply:
I tested your solution and the chained `replace` is faster than a single `replace` with a RegExp. Nice!

I also added an extra check to only re-write the files with that code instead of touching all the generated `.mjs` files.